### PR TITLE
add minor clarifications, removed tunr references, updated images to …

### PR DESCRIPTION
**Pull request by superbuggy**
…match grumblr rails api

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
 - Describe what an API is, and why we might use one.
-- Describe the purpose and syntax of `respond_to`
+- Use Postman to test an API.
+- Describe the purpose and syntax of `respond_to`.
 - Make a Rails app that provides a JSON API.
 
 ## Framing
 
-Yesterday afternoon you learned how to use Angular and `ngResource` to communicate with an API. In particular, you performed CRUD actions on a Grumblr API the instructors made for you. This morning you will learn how to build a Rails API from the ground up and create a back-end that serves up JSON along with the usual HTML views.
+In this lesson you will learn how to build a Rails API from the scratch and create a back-end that serves up JSON, in addition the usual HTML views. We will be focusing almost entirely on the back-end itself. To test our API, we will be using a very useful program called Postman.
```
Comment (AndyWhitley) [inline readme.md]: `from scratch` not `from the scratch`

File: readme.md [hunk 2]
```diff
@@ -20,7 +21,7 @@ Yesterday afternoon you learned how to use Angular and `ngResource` to communica
 </details>
 
 <details>
-  <summary><strong>How can we go about accessing an API programmatically?</strong></summary>
+  <summary><strong>How can we go about accessing an API within our programs?</strong></summary>
 
   > Using jQuery's AJAX method, Angular's ngResource or some other equivalent.
```
Comment (AndyWhitley) [inline readme.md]: Maybe mention Axios?

> Using jQuery's AJAX method, Angular's ngResource, Node's Axios library or some other equivalent.

File: readme.md [hunk 3]
```diff
@@ -37,7 +38,7 @@ Yesterday afternoon you learned how to use Angular and `ngResource` to communica
```
Comment (AndyWhitley) [inline readme.md:38]: Add comma to line 33:
we need to provide a `url`, `type` (HTTP method) and `dataType`

File: readme.md [hunk 4]
```diff
@@ -50,6 +51,8 @@ Form pairs and explore the API links in the below table. Record any observations
 | **[StarWars](http://swapi.co/)** | http://swapi.co/api/people/3 |
 | **[Stocks](http://dev.markitondemand.com/MODApis/)** | http://dev.markitondemand.com/Api/Quote/json?symbol=AAPL |
 
+The raw, unformatted JSON output that comes back from an API can be formatted by [this Chrome Extension, JSON Formatter](https://chrome.google.com/webstore/detail/json-formatter/bcjindcccaagfpapjjmafapmmgkkhgoa?hl=en)
+
 ## A Closer Look at an API Request (5 minutes / 0:15)
 
 Let's make a basic HTTP request to an API. While we can do this in the browser, we're going to use Postman - a Chrome plug-in for making HTTP requests - so we can not only look at it in more detail, but also make `POST` `PUT` and `DELETE` from the browser without building an app.
```
Comment (AndyWhitley) [inline readme.md:58]: replace definition of Postman as a 'plug-in' to an 'desktop application'

File: readme.md [hunk 5]
```diff
@@ -3,14 +3,15 @@
...
+- Describe the purpose and syntax of `respond_to`.
 - Make a Rails app that provides a JSON API.
 
 ## Framing
 
-Yesterday afternoon you learned how to use Angular and `ngResource` to communicate with an API. In particular, you performed CRUD actions on a Grumblr API the instructors made for you. This morning you will learn how to build a Rails API from the ground up and create a back-end that serves up JSON along with the usual HTML views.
+In this lesson you will learn how to build a Rails API from the scratch and create a back-end that serves up JSON, in addition the usual HTML views. We will be focusing almost entirely on the back-end itself. To test our API, we will be using a very useful program called Postman.
 
-## A Quick Refresher (5 minutes / 0:05)
+## Introducing APIs (5 minutes / 0:05)
```
Comment (AndyWhitley) [inline readme.md]: Change header from 'Introducing APIs' to just 'APIs'

File: readme.md [hunk 6]
```diff
@@ -81,13 +84,15 @@ Let's demonstrate using Grumblr. Clone down this [starter code](https://github.c
...
+$ bundle install
+$ rails db:create db:migrate db:seed
 $ rails s
 ```
 
 > The solution to today's code is available on the `api-solution` branch
 
-Earlier we used an HTTP request to retrieve information from a 3rd party API. Under the hood, that API received a GET request in the exact same way that the Rails application we have build in class thus far have received GET requests.
-* All the requests that our Rails application can receive are listed when we run `rake routes` in the Terminal. We create RESTful routes and corresponding controller actions that respond to `GET` `POST` `PATCH` `PUT` and `DELETE` requests.
+Earlier we used an HTTP request to retrieve information from omdbapi.com, a 3rd party API. Under the hood, that API received a GET request in the exact same way that the Rails application we have build in class thus far have received GET requests.
```
Comment (AndyWhitley) [inline readme.md]: Earlier we used Postman to send an HTTP request to retrieve...

**Comment by AWhitleyGA** [comment]
left comments

**Pull request merged**